src/status_im/common/enter_seed_phrase/view.cljs

```diff
@@ -96,12 +89,24 @@
                                                            "keyboardDidHide"
                                                            #(reset! keyboard-shown? false))
                      invalid-seed-phrase?    (reagent/atom false)
+                     input-ref               (reagent/atom nil)
+                     on-input-ref            (fn [ref]
```

Comment:
`on-input-ref` sounds a little unusual to me. Why not pass the ref as a prop `:ref #(reset! input-ref %)`

Reply:
Yeah its an odd name, but we're passing it to a component named `recovery-phrase-form` and I was trying to make it explicit that we're expecting that component to use `on-input-ref` as the `ref` callback for it's internal input component.

That being said, I'll see if I can revise this to use the `ref` prop instead, but I'm not sure if that will be clear which ref is being captured.